tests/conftest.py

```diff
@@ -618,3 +618,22 @@ def teardown_packages_update():
             assert result["rc"] == 0
 
     request.addfinalizer(teardown_packages_update)
+
+
+@pytest.fixture(scope="function")
+def setup_rpmsave_file(request, ansible_module):
+    """This fixture is used to create a .rpmsave file for test
```

Comment:
```suggestion
    """This fixture creates an .rpmsave file.
```
As fixtures can be used by more than one test I dont think you should mention test name here.